src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Alloying.java

```diff
@@ -74,6 +83,7 @@ public boolean removeByInputs(FluidStack... inputs) {
         return false;
     }
 
+    @MethodDescription(type = MethodDescription.Type.REMOVAL)
```

Comment:
`type = MethodDescription.Type.REMOVAL` is the default, so this can just be annotated `@MethodDescription`